projects/wipp-frontend-app/src/environments/environment.prod.ts

```diff
@@ -0,0 +1,26 @@
+import {version} from '../../../../package.json';
+
+export const environment = {
+  production: true,
+  version: version,
+  apiRootUrl: '/api',
+  keycloak: {
+    url: '@KEYCLOAK_URL_VALUE@',
+    realm: 'WIPP',
+    clientId: 'wipp-public-client'
+  },
+  uiPaths: {
+    csvCollectionsPath: 'csv-collections',
+    genericDatasPath: 'generic-datas',
```

Comment:
Agreed! The main reason I chose `genericDatas/generic-datas` was to avoid having a long name. But it might be confusing more than anything else. It would take a bit of time to do the renaming indeed but it would be better to do it now. The name has to change in WIPP-backend as well.